### DISCUSSION TITLE
Suggest a new feature

### DISCUSSION THREAD

**Discussion by ngdevc** [twbs/bootstrap#35681] [Q&A] [answered]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

I need the bottom sheet for mobile responsive. when I use material UI, I used the bottom sheet, but currently, I can't find it in bootstrap.
Thank you.

### Motivation and context

In mobile responsive, the bottom sheet is used well, so It's needed when we build web applications with bootstrap.
thank you.

![image](https://user-images.githubusercontent.com/65579678/148803140-4fdb47ac-50ec-478b-a122-06058ed77a5e.png)

**Comment by kainers1** [comment, marked as answer]
Have you looked at offcanvas yet?
https://getbootstrap.com/docs/5.1/components/offcanvas/#placement

**Reply by ngdevc** [reply]
Thank you.

**Reply by ngdevc** [reply]
I have solved it.
